.github/dependabot.yml

```diff
@@ -24,6 +24,7 @@ updates:
       include: 'scope'
     versioning-strategy: increase
     ignore:
+      - dependency-name: "@angular/*"
```

Comment:
Can we just ignore major versions? Just in case there are any useful minor or patch fixes that go out.

Reply:
Good shout! I've updated to only ignore major versions